src/tasks/block.rs

```diff
@@ -113,15 +119,21 @@ impl InProgressBlock {
 /// BlockBuilder is a task that periodically builds a block then sends it for signing and submission.
 pub struct BlockBuilder {
     pub config: BuilderConfig,
+    pub ru_provider: crate::config::Provider,
```

Comment:
can replace these fully-qualified paths with an import?

Reply:
the compiler was complaining because we import 
```
use alloy::providers::Provider;
```
so it was necessary to disambiguate

however i don't see in the code where that import is being used.. let me try to remove it and swap it

Reply:
https://github.com/init4tech/builder/pull/31/commits/7371efa93700c4855e1505bf6f7c976ebf627baf